backend/manager/modules/bll/src/main/java/org/ovirt/engine/core/bll/storage/disk/image/TransferDiskImageCommand.java

```diff
@@ -136,6 +137,9 @@
     @Inject
     private ResourceManager resourceManager;
 
+    @Inject
+    private VdsDaoImpl vdsDaoImpl;
```

Comment:
Seems to me there is some code missing here
Also, you should use DAO object with interface type, not the implementation.